osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/interfaceclass/attribute/MbusClientAttribute.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ * Copyright 2012-20 Fraunhofer ISE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file was originally part of jDLMS, where it was part of a group of classes residing in packages
+ * org.openmuc.jdlms.interfaceclass, org.openmuc.jdlms.interfaceclass.attribute and org.openmuc.jdlms.interfaceclass.method
+ * that have been deprecated for jDLMS since version 1.5.1.
+ *
+ * It has been copied to the GXF code base under the Apache License, Version 2.0 with the permission of Fraunhofer ISE.
+ * For more information about jDLMS visit
+ *
+ * http://www.openmuc.org
+ */
+package org.opensmartgridplatform.dlms.interfaceclass.attribute;
+
+import org.opensmartgridplatform.dlms.interfaceclass.InterfaceClass;
+
+/**
+ * This class contains the attributes defined for IC MbusClient.
+ */
+public enum MbusClientAttribute implements AttributeClass {
+
+    LOGICAL_NAME(1),
+    MBUS_PORT_REFERENCE(2),
+    CAPTURE_DEFINITION(3),
+    CAPTURE_PERIOD(4),
+    PRIMARY_ADDRESS(5),
+    IDENTIFICATION_NUMBER(6),
+    MANUFACTURER_ID(7),
+    VERSION(8),
+    DEVICE_TYPE(9),
+    ACCESS_NUMBER(10),
+    STATUS(11),
+    ALARM(12),
+    CONFIGURATION(13),
+    ENCRYPTION_KEY_STATUS(14);
+
+    static final InterfaceClass INTERFACE_CLASS = InterfaceClass.MBUS_CLIENT;
+
+    private MbusClientAttribute(final int id) {
+        this.id = id;
+    }
+
+    private int id;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1213)